cli/core.py

```diff
@@ -307,6 +307,8 @@ def dockerized(c, engine):
                 "LAMBDA_SECRET_ACCESS_KEY": creds.secret_key,
                 "LAMBDA_SESSION_TOKEN": creds.token,
             }
+        else:
+            raise Exit(message=error.response['Error']["Message"], code=1)
```

Comment:
are you sure all the error responses have the `Error` and `Message` field?

Looking at https://github.com/boto/botocore/blob/fec0e5bd5e4a9d7dcadb36198423e61437294fe6/botocore/exceptions.py#L526-L537
It would seem safer to use ClientError.msg

Reply:
your probably right changing this now

Reply:
- could you provide a link along with the documentation snippet so that the reader can have the context if he wants to know more 🔍 
- I would always remain careful about the structure of what is returned by an API, especially if you are already in an error handling context where no more `except` is going to be there to handle the exception. If you look at the snippet I linked above, Boto themselves are careful about what is returned by AWS 😅 
```
error = error_response.get('Error', {})
error_message=error.get('Message', 'Unknown')
```
- `msg is not attached to self` -> you are right, you can get it with str(error) though as it is used in the super constructor
```
super().__init__(msg)
```